stdpopsim/qc/HomSap.py

```diff
@@ -1747,3 +1747,46 @@ def Kim2017():
 
 
 _species.get_dfe("Gamma_K17").register_qc(Kim2017())
+
+
+def genericDFE():
+    """
+    gamma distribution based on Kyriazis et al. 2022
+    w/ discrete dominance coefficients
+    and lethal mutations
+    """
+
+    neutral = stdpopsim.MutationType()
+    gamma_mean = -0.0131
+    gamma_shape = 0.186
+    h_values = [0.45, 0.2, 0.05, 0]
+    h_breaks = [0.001, 0.01, 0.1]
+    negative = stdpopsim.MutationType(
+        dominance_coeff_list=h_values,
+        dominance_coeff_breaks=h_breaks,
+        distribution_type="g",
+        distribution_args=[gamma_mean, gamma_shape],
+    )
+    lethal = stdpopsim.MutationType(
+        distribution_type="f",
+        distribution_args=[-1],
+        dominance_coeff=0,
+    )
+
+    ns_proportion = 2.31 / (2.31 + 1.0)
+    lethal_proportion = 0.003 * ns_proportion
+    ns_proportion = ns_proportion - lethal_proportion
+    return stdpopsim.DFE(
+        id=id,
```

Comment:
I think the error is because you haven't defined `id`, and
```
$ python3
Python 3.13.1 (main, Jan  3 2025, 10:26:34) [GCC 14.2.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> id
<built-in function id>
```